com.ampl.ide.yaml

```
@@ -0,0 +1,46 @@
+app-id: com.ampl.ide
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: amplide.sh
+tags : 'proprietary'
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-documents
+modules:
+  - name: amplide
+    buildsystem: simple
+    build-commands:
+      - install -D amplide.sh /app/bin/amplide.sh
+      - install -Dm 755 apply_extra /app/bin/apply_extra
+      - install -Dm644 -t /app/extra/metainfo com.ampl.ide.metainfo.xml
```

this is not the proper path

```suggestion
      - install -Dm644 -t /app/share/metainfo com.ampl.ide.metainfo.xml
```